tests/config_test.go

```diff
@@ -39,6 +42,33 @@ import (
 var _ = Describe("[Serial][rfe_id:899][crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-compute]Config", func() {
 
 	var virtClient kubecli.KubevirtClient
+	var CheckIsoVolumeSizes func(vmi *v1.VirtualMachineInstance)
+
+	tests.BeforeAll(func() {
```

Comment:
This `tests.BeforeAll` looks like a leftover. Can it be that you simply want

```
var CheckIsoVolumeSizes func(vmi *v1.VirtualMachineInstance) = func(vmi *v1.VirtualMachineInstance) { [...]}
```

Reply:
I can do that. I was trying to stick to the pattern shown in the other test file that was changed. Some of those functions might have needed initialization but this one doesn't.

Reply:
All `BeforeAll` usages should be removed from the test files to avoid issues with parallel runs. In this case you don't have to replace it with a `BeforeEach`, you can just directly assign it from my perspective.